src/basic/lines.js

```diff
@@ -14,19 +14,30 @@
 	EXTERNAL DEPENDENCIES
  */
 
-import {curry, compose, not, equals, identity, cond} from 'ramda';
+const ramda = require('ramda');
```

Comment:
yep, I know but es6 modules broke browserify when we link directly the source in the `main` package. 

(the idea is to enable-hot reloading in parent packages when a change in children is detected to improve development speed)